src/main/java/com/matt/forgehax/mods/StepMod.java

```diff
@@ -91,6 +156,11 @@ public void onPacketSending(PacketEvent.Outgoing.Pre event) {
 
   @Override
   public String getDisplayText() {
-      return String.format("%s[%s]", super.getDisplayText(), stepHeight.get().toString());
+    return String.format(
```

Comment:
If you must do this, please change the override from getDisplayText() to getDebugDisplayText()

Reply:
You hate it that much? Well then.